src/coreclr/System.Private.CoreLib/src/System/Runtime/ExceptionServices/AsmOffsets.cs

```diff
@@ -16,9 +16,9 @@ class AsmOffsets
     // Debug build offsets
 #if TARGET_AMD64
 #if TARGET_UNIX
-    public const int SIZEOF__REGDISPLAY = 0x1a90;
-    public const int OFFSETOF__REGDISPLAY__SP = 0x1a78;
-    public const int OFFSETOF__REGDISPLAY__ControlPC = 0x1a80;
+    public const int SIZEOF__REGDISPLAY = 0x1b90;
```

Comment:
I'm not sure how these values are calculated, the values are updated based on the error massage, so want to double check with the reviewers.

Reply:
These constants are what the sizeof(...) / offsetof(...) would get during C/C++ compilation. The offsets above are verified to match with the real values during the C/C++ compilation phase, so if the runtime build passes, you are safe.

Reply:
Thanks for the explanation!